src/components/Tabs/verticalFlatTabs.css

```diff
@@ -0,0 +1,33 @@
+.lil-vertical-flat-tabs{
+    @apply lil-list-none lil-overflow-x-auto;
+}
+
+.lil-vertical-flat-tabs > .lil-flat-tab{
+    @apply lil-bg-white lil-opacity-50 lil-py-5 lil-px-0 lil-outline-none lil-flex lil-whitespace-normal lil-cursor-pointer;
+}
+
+.lil-vertical-flat-tabs .lil-flat-tab-active{
+    @apply lil-opacity-100 lil-bg-blue-50;
+}
+
+.lil-flat-tab-active {
+    position: relative;
+    @import  lil-bg-grey-190;
```

Comment:
any reason for using `@import` ? y not `@apply` ?

Reply:
for the pseudo-selector like :hover, :active, :before to access this class it should be defined as @import , in this case lil-flat-tab-active:before uses this attribute